tests/kafkatest/sanity_checks/test_kafka_version.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.tests.test import Test
+
+from kafkatest.services.zookeeper import ZookeeperService
+from kafkatest.services.kafka import KafkaService
+from kafkatest.services.kafka import KafkaVersion
+
+
+class KafkaVersionTest(Test):
```

Comment:
That's a good question, I'm not sure I'm clear either.

Why we (at least I) want them - it is very helpful to have a suite of tests that is relatively quick to run, and can be run repeatedly while making updates to service classes to catch errors and regressions. Analogous to unit tests.

With that in mind, when should a test be a sanity check?
- Should we be disciplined in keeping sanity focused, each on on a particular service class, and integrating a minimum of other services?
- Or, simply a test which is "quite quick" to run?

Having written this, I'd lean toward the former, but it's all pretty hand-wavy